src/views/CardStack/CardStack.js

```diff
@@ -94,7 +94,7 @@ class CardStack extends React.Component<Props, State> {
   componentWillReceiveProps(nextProps: Props) {
     if (nextProps.statusBarSize !== this.props.statusBarSize) {
       this.setState({
-        headerHeight: (this.props.isIOS ? 45 : 41) + this.props.statusBarSize,
+        headerHeight: (this.props.isIOS ? 45 : 41) + nextProps.statusBarSize,
```

Comment:
This looks like it's a bug fix so it's good to merge.

Reply:
@bluhmann Did you check the base fork to see if this has been fixed already?

Reply:
I don't see it, but there structure is a bit different from ours, they don't have any `Card` prefixes. From the best I could tell though i didn't see a change like this.

Reply:
It looks like something we added in, this was added 3 months ago.